sources/packages/backend/apps/workers/src/controllers/application/application.controller.ts

```diff
@@ -118,12 +129,31 @@ export class ApplicationController {
         application.data,
       );
       if (exceptions.length) {
-        const createdException =
-          await this.applicationExceptionService.createException(
-            job.variables.applicationId,
-            exceptions,
+        let createdException: ApplicationException;
```

Comment:
There is no need to create this externally to the transaction. The return can be from the `job.complete`, right?